lua/entities/gmod_tardis/cl_init.lua

```diff
@@ -2,4 +2,5 @@ include('shared.lua')
 
 ENT:AddHook("PlayerInitialize", "interior", function(self)
     self.metadata=TARDIS:GetInterior(net.ReadString())
+    self.metadata = TARDIS:MergeInteriorTemplates(self.metadata, true, self:GetCreator(), self)
```

Comment:
Could this not be done at the stage of TARDIS:GetInterior, perhaps by passing a couple extra parameters into that? Same for all other calls of this function

Reply:
Good idea
Done